amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -67,6 +88,20 @@ function submitClean(){
 	document.aimAuditLoggerManagerForm.submit();
 }
 
+function submitFilter() {
+	if (document.getElementById("userId").value != null || document.getElementById("teamId").value != null) {

```

Comment:
you dont need to set the values to filter user, if you comment the code and just submit, values are present in their properties.